README.md

```diff
@@ -24,6 +24,13 @@ Add the package to `app/AppKernel.php`, *before* the EzPublishCoreBundle declara
 
 The package will replace the services from the kernel, thus enabling the new features, such as multi-tagging.
 
+The application cache class needs to be customized. You can do so by setting the `SYMFONY_HTTP_CACHE_CLASS`
+environment variable:
+
+    export SYMFONY_HTTP_CACHE_CLASS='EzSystems\PlatformHttpCacheBundle\AppCache'
+
+Do not forget to restart your web server.
```

Comment:
Keep it simple: "Alternatively, if you need to make your own customizations, you should extend this class and set `SYMFONY_HTTP_CACHE_CLASS` accordingly."